compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/animation/Animation.skiko.kt

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2024 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package androidx.compose.ui.animation
+
+import androidx.compose.runtime.withFrameNanos
+import kotlin.math.min
+import kotlin.time.Duration
+import kotlin.time.Duration.Companion.nanoseconds
+
+internal fun easeInOutTimingFunction(progress: Float) = if (progress < 0.5) {
+    2 * progress * progress
+} else {
+    (-2 * progress * progress) + (4 * progress) - 1
+}
```

Comment:
Mixing ints, doubles and floats. Let's align that

Reply:
Done